samples/openapi3/client/petstore/dart-dio/petstore_client_lib_fake/lib/model/enum_class.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:built_collection/built_collection.dart';
+import 'package:built_value/built_value.dart';
+import 'package:built_value/serializer.dart';
+
+part 'enum_class.g.dart';
+
+class EnumClass extends EnumClass {
```

Comment:
There is something wrong here when these types get generated - this was fixed in 7fae4ab4116c826ca64921babd239fbf1eb2d180

Reply:
yep just fixing that :)

Reply:
You need to delete the 4 old files manually.